_posts/2020-08-14-status-update.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "14th August 2020: IAC Team Status Update"
+author: pmcmaw
+categories:
+  - team
+  - status
+tags:
+  - releases
+  - shoutouts
+  - puppet7
+  - apt
+  - kubernetes
+  - docker
+  - language
+  - mysql
+  - registry
+---
+
+The IAC team has been working on a lot of things over the past week.
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+We have been busy this week with releasing the following modules:
+- [puppetlabs/docker](https://forge.puppet.com/puppetlabs/docker) (v3.11.0)
+- [puppetlabs/apt](https://forge.puppet.com/puppetlabs/apt) (v7.5.0)
+- [puppetlabs/registry](https://forge.puppet.com/puppetlabs/registry) (v3.1.1)
+- [puppetlabs/mysql](https://forge.puppet.com/puppetlabs/mysql) (v10.7.0)
+
+
+## Community Contributions
+A massive thanks to [scoopex][scoopex] for a number of enhancements and fixes to [puppetlabs-kubernetes][puppetlabs-kubernetes]. There are a lot of fixes and improvements incorporated in each of these PRs - for more details, take a look at the commits outlining each fix:
+- [`puppetlabs-kubernetes#418`][kubernetes-pr-418] resolves issues [#365][kubernetes-issue-365] and [#415][kubernetes-issue-415]
+- [`puppetlabs-kubernetes#420`][kubernetes-pr-420] resolves issues [#190][kubernetes-issue-190] and [#417][kubernetes-issue-417]
+
+Thanks to [tphoney][tphoney] for [this enhancement][abs-enhancement] to the ABS provision module!
+
+Shout out to Jason Stewart from our Customer Support team for liasing with [Adrian][adrian_iurca], [MODULES-10734](https://tickets.puppetlabs.com/browse/MODULES-10734) has now been resolved and released.
+
+
+## Puppet 7
+Puppet 7 testing is ongoing. [Ciaran][ciaran_mccrisken] is kicking Puppet 7 testing off when the nightly builds are published.
+
+
+## Terminology changes
+The team is continuing to work on this. With the amount of modules and tools the team looks after this is quite a large piece of work that we are dedicated to completing in a timely matter. Currently [David Swan][david_swan] is continuing to work on this to get it over the line.
```

Comment:
I'm thinking if we can add here a small introduction, something like "how was mentioned in last week blog post the team is ... " or kind of :-??

Reply:
Good shout 👍 Added this in.